doc/release-notes.md

```diff
@@ -20,3 +20,20 @@ inputs each. Starting from this release, `-mempooltxinputlimit` will be enforced
 before the Overwinter activation height is reached, but will be ignored once
 Overwinter activates. The option will be removed entirely in a future release
 after Overwinter has activated.
+
+`NODE_BLOOM` service bit
+------------------------
+
+Support for the `NODE_BLOOM` service bit, as described in [BIP
+111](https://github.com/bitcoin/bips/blob/master/bip-0111.mediawiki), has been
+added to the P2P protocol code.
+
+BIP 111 defines a service bit to allow peers to advertise that they support
+bloom filters (such as used by SPV clients) explicitly. It also bumps the protocol
```

Comment:
"Bloom filters" and "Bloom filtering".

(Named for Burton Howard Bloom. [BIP 37](https://github.com/bitcoin/bips/blob/master/bip-0037.mediawiki) correctly capitalizes it, although [BIP 111](https://github.com/bitcoin/bips/blob/master/bip-0111.mediawiki) does not.)